wrappers/python/simtk/openmm/app/forcefield.py

```diff
@@ -989,8 +990,8 @@ def _buildBondedToAtomList(self, topology):
 
         Returns
         -------
-        bondedToAtom : list of set of int
-            bondedToAtom[index] is the set of atom indices bonded to atom `index`
+        bondedToAtom : list of set list int
```

Comment:
Shouldn't this be

```suggestion
        bondedToAtom : list of list of int
```

?

Reply:
Thanks, fixed.